luma/core/interface/serial.py

```diff
@@ -103,14 +103,25 @@ def data(self, data):
         :type data: list, bytearray
         """
         if self._managed:
-            self._bus.i2c_rdwr(self._i2c_msg_write(self._addr, [self._data_mode] + data))
+            block_size = 4096
```

Comment:
can you add a comment for `block_size` describing why this magic nr is needed?

Reply:
Added comment - I started out with a block size of 1K as the ioctl docs alluded to, then kept doubling it until it no longer worked. 4K is the last block size that doesn't fail on RPi. It seems a good compromise between transferring large chunks and fewer calls.